[23.05]: mac80211: Update to version 6.1.145-1

hauke opened the pull request:
This updates mac80211 to version 6.1.145-1. This code is based on Linux 6.1.145 and contains all fixes included in the upstream wireless subsystem from that kernel version. This includes many bugfixes and also some security fixes.

The removed patches were applied upstream.

namiltd commented:
Is there a chance to already adapt the package to kernel 6.12-rc7? Stable version will be any day now.

hauke committed:
mac80211: Update to version 6.1.145-1

This updates mac80211 to version 6.1.145-1. This code is based on Linux
6.1.145 and contains all fixes included in the upstream wireless
subsystem from that kernel version. This includes many bugfixes and also
some security fixes.

The removed patches were applied upstream.

Link: https://github.com/openwrt/openwrt/pull/16948
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
